docs/recommendations/carousel/carousel.md

```diff
@@ -0,0 +1,72 @@
+# Carousel Props
+
+* [component](#component)
+* [loaderComponent](#loadercomponent)
+* [getComponentProps](#getcomponentprops)
+
+### component
+The props for your component comes from [getComponentProps](#getcomponentprops). This component renders each item in the carousel.
```

Comment:
I think it would be a little more clear if it was something along these line.

```suggestion
A custom component that will be used to render a single item in the carousel. The props for this component are determined by [getComponentProps](#getcomponentprops).
```